tools/nni_cmd/launcher_utils.py

```diff
@@ -253,6 +253,12 @@ def validate_pai_trial_conifg(experiment_config):
         experiment_config['trial']['shmMB'] > experiment_config['trial']['memoryMB']:
             print_error('shmMB should be no more than memoryMB!')
             exit(1)
+        #backward compatibility
+        warning_information = '{0} is not supported in NNI anymore, please remove the field in config file!'
```

Comment:
could add a little more information, for example, "please refer to the link for the practices of how to get data and output model in trial code".

Reply:
updated.